tests/test_checker/test_noqa.py

```diff
@@ -93,6 +93,7 @@
         'WPS239': 1,
         'WPS240': 0,  # only triggers on 3.12+
         'WPS241': 1,
+        'WPS242': 0,
```

Comment:
please, add at least one to `noqa.py`. we need this to:
- make sure that `noqa` works for the correct node
- the whole integration process worked

Reply:
oh, i`ve missed that, sorry.
Fixed